CatalogAPI/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using CatalogAPI.Data;
+using CatalogAPI.Data.Model;
+using CatalogAPI.DTO;
+using CatalogAPI.Extensions;
+using Microsoft.AspNetCore.Mvc;
+namespace CatalogAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ItemsController : ControllerBase
+    {
+        [HttpGet("categoryId/{categoryId}/page/{page}/pageSize/{pageSize}")]
```

Comment:
I don't think page, pageSize is a good resource. Is it even a resource? consider /api/categories/{id}items + filtering or /api/items/ with filters as queryparams or in the body

Reply:
I changed it with model.